trestle/core/commands/author/profile.py

```diff
@@ -220,7 +261,14 @@ def _replace_modify_set_params(profile: prof.Profile, param_str_dict: Dict[str,
 
     @staticmethod
     def assemble_profile(
```

Comment:
Related to other static methods as well, I think this is where the disadvantages of static methods show up the most. Half of those arguments repeat throughout the code making function signatures bulky, if we ever consider to refactor those classes, I think it would be much cleaner if the parameters are set in the class constructor instead.